src/factories/eventData.ts

```diff
@@ -28,7 +27,7 @@ export async function eventDataFactory(
     }
 
     if (!runGraphData.end_time) {
-      interval = setTimeout(() => start(), getIntervalForDataSize(runGraphData))
+      interval = setTimeout(() => start(), config.fetchEventsInterval)
```

Comment:
out of curiosity - should this be `setInterval`? The `clearInterval` on line 36 isn't doing anything. It won't throw an error because `setTimeout` and `setInterval` take the same argument (a number id)

Reply:
Its using setTimeout because its calling itself. Basically after every fetch we evaluate if we should continue fetching and if we should we schedule another fetch.

Reply:
You're right about the clearInterval it should be clearTImeout. Will update